OWNERS

```diff
@@ -3,25 +3,22 @@
 filters:
   ".*":
     reviewers:
-    - bparees
     - deads2k
-    - sjenning
-    - smarterclayton
+    - sttts
     - soltysh
-    - tbielawa
+    - mfojtik
+    - marun
+    - tnozicka
+
+    # approvers are limited to to the team that manages rebases and pays the price for carries that are introduced
```

Comment:
let's call this one out explicitly so it isn't overlooked. if something is on fire and none of us is reachable, architects can edit the labels

Reply:
:+1: